packages/aws-lambda/src/triggers.js

```diff
@@ -62,24 +62,50 @@ exports.enrichSpanWithTriggerData = function enrichSpanWithTriggerData(event, co
 };
 
 function isApiGatewayProxyTrigger(event) {
-  // Note: An application load balancer event also has event.path and event.httpMethod but it does not have
+  // NOTE: An application load balancer event also has event.path and event.httpMethod but it does not have
   // event.resource.
-  return event.resource != null && event.path != null && event.httpMethod != null;
+  return (
+    (event.resource != null && event.path != null && event.httpMethod != null) ||
+    (event.rawPath && event.version === '2.0')
```

Comment:
FYI: no other trigger has this format. See https://docs.aws.amazon.com/lambda/latest/dg/lambda-services.html